builder/vsphere/common/step_export.go

```diff
@@ -59,13 +59,15 @@ import (
 // ./output_vsphere/example-ubuntu.ovf
 // ```
 type ExportConfig struct {
-	// name of the ovf. defaults to the name of the VM
+	// Name of the ovf. defaults to the name of the VM
 	Name string `mapstructure:"name"`
-	// overwrite ovf if it exists
+	// Overwrite ovf if it exists
 	Force bool `mapstructure:"force"`
-	// include iso and img image files that are attached to the VM
+	// Deprecated: Images will be removed in a future release. Please see `image_files` for more details on this argument.
 	Images bool `mapstructure:"images"`
-	// generate manifest using sha1, sha256, sha512. Defaults to 'sha256'. Use 'none' for no manifest.
+	// Include additional image files that are attached to the VM, such as nvram, iso, img.
+	ImageFiles bool `mapstructure:"image_files"`
```

Comment:
Do we want to be more precise on where is this going to include them ? For example:
```suggestion
	// In exported files, include additional image files that are attached to the VM, such as nvram, iso, img.
	ImageFiles bool `mapstructure:"image_files"`
```